README.md

```diff
@@ -15,6 +15,13 @@ git clone git@github.com:shardeum/validator-cli.git
 cd validator-cli
 ```
 
+The validator CLI typically requires a symlink to the Shardeum repo at a specific path. If you're running the CLI manually, ensure the symlink is set up correctly:
+
+```bash
+ln -s /path/to/shardeum/repo ../validator
```

Comment:
add a `cd cli` here so that the relative path of `../validator` in the next commands works correctly